library/src/main/java/com/novoda/downloadmanager/DownloadManagerRequestCreator.java

```diff
@@ -0,0 +1,26 @@
+package com.novoda.downloadmanager;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class DownloadManagerRequestCreator {
```

Comment:
This class now holds of the requests that the app does 🤔  at least I'm pretty sure it does, unless I missed any 😆

Reply:
What about renaming this to `NetworkRequestCreator`, as this object only leaves in the specific implementation `NetworkFileDownloader` of the interface `FileDownloader`